includes/definitions/avr-hd.yaml

```diff
@@ -0,0 +1,9 @@
+os: avr-hd
+text: 'avr-hd'
+type: appliance
+icon: exterity
+over:
+    - { graph: device_bits, text: Traffic }
+discovery:
+    - sysDescrId:
```

Comment:
SNMPwalk here: 
http://pastebin.com/raw/Rem5nv9d

Don't think that's possible?